gotradepal/backend/app/core/config.py

```
@@ -0,0 +1,11 @@
+from pydantic_settings import BaseSettings
+
+class Settings(BaseSettings):
+    database_url: str = "sqlite:///./gotradepal.db"
+    redis_url: str = "redis://localhost:6379/0"
+    jwt_secret: str = "dev-secret"
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to: "jwt_secret"

(S105)